src/HttpKernel/ContaoKernel.php

```diff
@@ -51,10 +70,22 @@ public function registerBundles()
     /**
      * {@inheritdoc}
      */
+    public function getProjectDir()
+    {
+        if (null === $this->projectDir) {
+            $this->projectDir = dirname(dirname(dirname(dirname(dirname(__DIR__)))));
```

Comment:
In the unit tests we could compare the project dir path against the path of the ContaoKernel.php file.